scripts/docker-helper/readme.md

```diff
@@ -177,3 +185,72 @@ If you REALLY get stuck and want to destroy _**ALL**_ docker containers/volumes/
 ```
 docker stop $(docker ps -q)&&docker system prune&&docker volume prune
 ```
+
+## cht-docker-compose.log
+
+This log will be output every time you call the script.  It will be created and appended to in the directory where you specified your environment file to be (`-e PATH/TO/env_file`). 
+
+There are three types of lines in this file. The first line will always be the Start line: `item="start"`. Then there will be a Status line: `item="status"`.  Finally, there will be two lines, one for each docker container: `item="docker_logs"`.
```

Comment:
Some feedback from my first time using this: I kind of expected there to be some kind of `item="end"` line in the logs even in the happy-path case.  I initially wondered (before reading the script code) if records would continue to be added to the log somehow for the lifetime of the containers.  (When first reading through the log file, I thought my containers had actually died since the logging just seemed to stop.)

Not really asking for any changes here, just mentioning my experience.

Reply:
No - I think this is a good point!  The whole idea of adding logging is to help tell what's going on - adding a terminal log entry is a good idea. Added this in fb3913c71